jsrc/jtype.h

```diff
@@ -274,70 +282,98 @@ typedef I SI;
 #define XDSIZE    sizeof(DX)
 #define XZSIZE    sizeof(ZX)
 
+//more shifts
+enum {
+    ASGNX = 21,
+    MARKX = 22,
+    NAMEX = 23,
+    SYMBX = 24,
+    CONWX = 25,
+    ADVX  = 26,
+    VERBX = 27,
+    LPARX = 28,
+    CONJX = 29,
+    RPARX = 30,
+    RESVX = 31,  // reserved so types can be I types
+                 // NOTE: maxtype & maybe others require bit 31 to be 0!!
+};
```

Comment:
These should technically be the same `enum` as the nouns :arrow_up: 

Also, can we name these enums?

Reply:
New commit merged some together.

I was thinking the enum name without a scope doesn't do much. Though I don't see a reason why we can't. What names you think? I guess nouns seems to be the entire first 20 bits. I'm not sure what to call the 21-31 bits. #naming_is_hard